src/parser/jobs/brd/modules/Sidewinder.js

```diff
@@ -56,6 +57,11 @@ export default class Sidewinder extends Module {
 			abilityId: ACTIONS.SIDEWINDER.id,
 		}, this._onSidewinderCast)
 
+		this.addHook('cast', {
+			by: 'player',
+			abilityId: ACTIONS.SHADOWBITE.id,
+		}, this._onShadowbiteCast)
+
```

Comment:
A simpler way to get the same result, and not have to track last Shadowbite cast time, would be to use the FFLogsEvents compatibilty shim instead, like core entities does: https://github.com/xivanalysis/xivanalysis/blob/8035dedd6b4361ac4fabee31a687c116cd3d9cf8/src/parser/core/modules/Entities.js#L119

This would have two advantages:
- Won't mysteriously break if the cast events are missing or if their timing changes relative to damage events
- Doesn't require hooking two sets of actions for each Shadowbite cast, so easier to maintain going forward.

Reply:
Thanks for the feedback. I went ahead and use the fflog event. Let me know if you want me to squash the commits into one or not.